src/Company.php

```diff
@@ -262,9 +262,9 @@ protected function init()
      * @return \TeamWorkPm\Response\Model
      * @throws \TeamWorkPm\Exception
      */
-    public function getAll()
+    public function getAll($request = null)
```

Comment:
please adjust the docheader as well

Reply:
👍

Reply:
well, null is the default but what is the expected type?

Reply:
updated to match the format in the rest of the project